synccit.user.js

```diff
@@ -242,7 +242,7 @@ function markLink(link) {
 	//element.className += '.synccit-read'; // adding the class doesn't seem to let it overwrite style even with !important
 										  // d'oh needed dot at front. replacing classname breaks RES 
 	if(element != null) { // seems on self post this will end up null or something. not sure why
-		element.style.opacity = .4;
```

Comment:
We could apply both sets of styling. I don't see why it has to be one or the other.

Reply:
The only issue I can see here is causing a conflict with Reddit gold where it highlights visited links. Could this cause issues?